src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/IdeCodeStyleOptions.cs

```diff
@@ -46,6 +46,7 @@ internal record class IdeCodeStyleOptions
     [DataMember] public CodeStyleOption2<bool> PreferConditionalExpressionOverAssignment { get; init; } = CodeStyleOption2.TrueWithSilentEnforcement;
     [DataMember] public CodeStyleOption2<bool> PreferConditionalExpressionOverReturn { get; init; } = CodeStyleOption2.TrueWithSilentEnforcement;
     [DataMember] public CodeStyleOption2<bool> PreferCompoundAssignment { get; init; } = CodeStyleOption2.TrueWithSuggestionEnforcement;
+    [DataMember] public CodeStyleOption2<bool> PreferInterpolatedString { get; init; } = CodeStyleOption2.TrueWithSuggestionEnforcement;
```

Comment:
This should be initialized in the constructor below. Please keep it after the `PreferCompoundAssignment` there as well